internal/services/databricks/databricks_virtual_network_peerings_resource_test.go

```diff
@@ -0,0 +1,335 @@
+	id, err := vnetpeering.ParseVirtualNetworkPeeringID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := clients.DataBricks.VnetPeeringsClient.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("making Read request on Databricks %s: %+v", *id, err)
+	}
+
+	return utils.Bool(resp.Model != nil), nil
+}
+
+func (DatabricksVirtualNetworkPeeringResource) update(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-databricks-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_virtual_network" "remote" {
+  name                = "acctest-vnet-%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  address_space       = ["10.0.1.0/24"]
+  location            = azurerm_resource_group.test.location
+}
+
+resource "azurerm_databricks_workspace" "test" {
+  name                = "acctest-ws-%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  sku                 = "standard"
+}
+
+resource "azurerm_databricks_virtual_network_peering" "test" {
+  name                = "acctest-%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_id        = azurerm_databricks_workspace.test.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+
+  allow_virtual_network_access = false
+}
+
+resource "azurerm_virtual_network_peering" "remote" {
+  name                      = "to-acctest-%[1]d"
+  resource_group_name       = azurerm_resource_group.test.name
+  virtual_network_name      = azurerm_virtual_network.remote.name
+  remote_virtual_network_id = azurerm_databricks_virtual_network_peering.test.virtual_network_id
+}
+`, data.RandomInteger, data.Locations.Primary)
+}
+
+func (DatabricksVirtualNetworkPeeringResource) requiresImport(data acceptance.TestData) string {
+	template := DatabricksVirtualNetworkPeeringResource{}.basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_databricks_virtual_network_peering" "import" {
+  name                = "acctest-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_id        = azurerm_databricks_workspace.test.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+}
+`, template, data.RandomInteger)
+}
+
+func (DatabricksVirtualNetworkPeeringResource) basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
+}
```

Comment:
this would indicate there's a bug in the resource, since things aren't being cleaned up properly, why's this needed?

```suggestion
provider "azurerm" {
  features {}
}
```

Reply:
Fixed.